src/components/ArticlesList.jsx

```diff
@@ -1,29 +1,67 @@
-import React, { useEffect, useState } from "react";
 import axios from "axios";
-import ArticleCard from "./ArticleCard";
-import {useParams} from "react-router-dom"
+import React from "react";
+import { useState, useEffect } from "react";
+import ArticleCard  from "./ArticleCard";
 
-export default function ArticlesList () {
-    const [articlesList, setArticlesList] = useState([])
-    
-    
-    useEffect(() => {
-        axios
-          .get("https://be-nc-news-tnfa.onrender.com/api/articles")
-          .then(({ data }) => {
-            console.log(data.articles)
-            setArticlesList(data.articles);
-          });
-      }, []);
+export default function topicsForm() {
+  const [topicsForm, setTopicsForm] = useState({
+    category_name: "",
+  });
 
-    return (
-        <div className="articles">
-            <h1>Articles:</h1>
-          {articlesList.map((article) => {
-            return <ArticleCard key={article.article_id} article={article} />;
-          })}
-                <br></br>
-        </div>
-      );
+  const [articlesList, setArticlesList] = useState([])
+
+  function handleChange(e) {
+    const { name, value } = e.target;
+    setTopicsForm({
+      ...topicsForm,
+      [name]: value,
+    });
+  const newURL = `http://localhost:5174/articles/topics/${value}`;
+  window.history.pushState(null, "", newURL);
```

Comment:
Unfortunately this method doesn't appear to work, there are errors being thrown in the console related to `window.history.pushState`. The URL you are using is the URL for your local server, which will be different when the site goes live. 

For the purposes of this ticket, and because you are using a dropdown style menu, my advice would be to discard the feature aiming to update the URL. This is always something you can revisit at a later point.